kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/screens/LibraryScreen.java

```diff
@@ -62,6 +66,8 @@
     private Caller<LibraryService> libraryService;
     private LibraryPlaces libraryPlaces;
     private SpaceContributorsListServiceImpl spaceContributorsListService;
+    private Event<NotificationEvent> notificationEvent;
+    private TranslationService ts;
```

Comment:
We should use whole names, later in code it can not be clear what "ts" means.
```suggestion
    private TranslationService translationService;
```

Reply:
@jakubschwan done!